### ISSUE TITLE
Web - Not Able to Activate the Last About Link with Keyboard

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#134]
When using a keyboard to tab through the About pop-up links, I can not tab to the last link and activate it with a keyboard because it closes. There might be some JavaScript code that limits the links to three that now needs to be changed after adding the new one a while back.

I assume the issue is on other platforms and languages, too, than the ones marked.

Please see the attached screenshot.

![ilab - comply chain - not able to activate the last About link with keyboard](https://user-images.githubusercontent.com/10423487/83899792-05bc2180-a727-11ea-9f5f-42de95130ead.png)

**Comment by jfc3-dol** [comment]
@Abychalil,
I can now get to and activate the last "About" pop-up link with a keyboard in IE 11 but not in Chrome and Firefox, and Edge is not even displaying the home page.

In IE 11, after activating the link, the "About" pop-up stays open.

**Comment by Abychalil** [comment]
Hi John, Thanks for reviewing this ticket. In chrome, IE and Edge I am able to open the 4th link via tabbing. 
Yes, About window stays open only  when we click on the 4th link. Our developers are working on it.  Let me work them and make sure that the 4th link can be opened in all browsers. 

And my question about Edge browser,  Comply Chain App cannot be opened in Edge? Are you getting a blank page when giving the link?  

Please let me know. We will fix that. 
@jfc3-dol

**Comment by jfc3-dol** [comment]
@Abychalil 

Thanks for the response.

Not sure why I can't tab to the last link and open it in some of those browsers.

Edge is showing a blank page. Not able to even see the home page.

Once after refreshing the page a few times, it showed up on my way out last night. Not sure if it's timing out somehow or not getting all the code it needs to render, or the refreshes finally get all the necessary files.

Let me try it again to see if I can get at least the home page to show.

**Comment by jfc3-dol** [comment]
@Abychalil 

At the moment, I can see the home page and others I randomly chose in English.

Who knows what's going on.

Once more updates are made for the next version, I will check things in Edge, too, when looking at the problems.

**Comment by jfc3-dol** [comment]
@Abychalil 

I can now open the last About pop-up link with a keyboard in IE11, Edge, and Chrome, but the pop-up stays open when the page loads. Please see the attached screenshot. I only took one showing Chrome.

For Firefox, I can now tab to the last About pop-up link and hit the enter key, but the page does not load as expected.

![ilab - about pop-up not closing](https://user-images.githubusercontent.com/10423487/88204363-25e16a80-cc19-11ea-90cd-abed0a055ffe.png)

**Comment by harpreetsdev** [comment]
@jfc3-dol 

A quick general question - Sandi requested a list of 508 issues for Comply Chain and Sweat and Toil on your behalf. I wanted to know if you need a list of all the "Fixed" 508 bugs for both apps OR a list of Outstanding bugs that we have in JIRA for you to match them against this Github list to make sure nothing is being left out.

Either would be ok, just need to make sure we provide the right list.

**Comment by harpreetsdev** [comment]
@jfc3-dol  

Also is there a way we could have a quick chat on some current Open issues? The reason for my request is that there are some issues in the "Open" list that should be closed, since they were fixed last Sprint, some have been fixed this Sprint waiting for your review (No rush on that). 

But then there are some issues in the "Grey Area" of Done vs Undone that have been fixed for the most part but not working under some specific circumstances. Such as - #55 where the Share button is working as expected in all browsers except IE11.
There are a few more like those.

What would be helpful is that those issues got closed and newer specific issues got created such as - "**Share button not working in IE11**". In addition to reducing bulk of outstanding issues in Github that'd also help us align Github issues 1:1 with the JIRA  issues that we use for internal progress tracking.  

Please let me know what you think.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17 

I'm busy reviewing lots of documents for a project that has to be done this week.

It might make sense that maybe we pull the remaining issues out of other issues, but will have to look at then which won't be until possibly the end of the week.

Might have time end of the week to talk.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

No problem at all. Take your time.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil 

Can someone e-mail me a list of the current open issues that probably should have new issues created because what is broken is not the issue started as the issue now?

It might make sense to have a call too.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

We are currently working on a list of open issues for Comply chain. It should be ready by sometime tomorrow. We'll forward that to you as soon as it's done.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil 

Thanks, will look forward to seeing them.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

Closing the issue, I'm now able to activate the last link in the About the list of links.

But will open a new issue for Chrome because the About list of links stays open only when you click the last items in the list but closes as expected in IE 11, Edge, and Firefox.
